### ISSUE TITLE
Function heading differs from prototype error

### ISSUE THREAD

**Issue by 21thyear** [openmultiplayer/open.mp#765]
**Describe the bug**
When you download the current release, and create a new (blank) game mode and compile it then, you'll get about one dozen bugs.

All errors are related to error 025: function heading differs from prototype. Actually, this is not what you want to see in the release, which is already the last one. This problem was not present on the RC version.

**To Reproduce**
1. Download release v1.1.0.2612
2. Open IDE, create a blank gamemode
3. Compile blank gamemode

**Expected behavior**
Successful compilation of a gamemod

**Screenshots and/or logs**
There's no need

**Release version**
[Release](https://github.com/openmultiplayer/open.mp/releases/tag/v1.1.0.2612)

**Comment by 21thyear** [comment]
The solution is simple, it's all easy, just want to get rid of such a thing in an official and up to date release
Error due to missing tags, in all publics where there is a weaponid, there is an error because there is no WEAPON tag. Same problem with in OnVehiclePaintJob paintjob-> paintjob-> paintjobid

In OnVehicleRespray(playerid, vehicleid, color1, color2) need to change, correct:
OnVehicleRespray(playerid, vehicleid, color1, color2)

**Comment by adib-yg** [comment]
This issue is in v1.1.0.2612

Fix:

* qawno/gamemode.new

https://pastebin.com/pQk3Kzgy

* qawno/filterscript.new

https://pastebin.com/t5DBFfTm


PR: https://github.com/openmultiplayer/qawno/pull/4

**Comment by 21thyear** [comment]
> This issue is in v1.1.0.2612
> 
> Fix:
> qawno/gamemode.new
> qawno/filterscript.new

Yes, it's fix

**Comment by AmyrAhmady** [comment]
Already fixed then

**Comment by adib-yg** [comment]
It still happens in v1.2.0.2670

**Comment by deprale** [comment]
happening with newest openmp

**Comment by StivenHL** [comment]
I also had a problem on line 256, which gave me error 025. Upon reviewing, I realized that this line is wrong when creating it from qawno.

original line:
public OnPlayerEditObject(playerid, playerobject, objectid, EDIT_RESPONSE:response, Float:fX, Float:fY, Float:fZ, Float:rotationX, Float:rotationY, Float:rotationZ)

Fixed line:
public OnPlayerEditObject(playerid, playerobject, objectid, EDIT_RESPONSE:response, Float:fX, Float:fY, Float:fZ, Float:fRotX, Float:fRotY, Float:fRotZ)

I don't know if I'm the only one this happens to you, anyway I'm posting it here, I don't speak English, so forgive me if I didn't make myself understood.
